Geneious/Geneious.download.recipe

```diff
@@ -54,7 +67,7 @@ See http://desktop-links.geneious.com/assets/installers/geneious/GeneiousVersion
 			<key>Arguments</key>
 			<dict>
 				<key>input_path</key>
-				<string>%pathname%/Geneious.app</string>
+				<string>%pathname%/%NAME%.app</string>
```

Comment:
won't this cause code signature verification to fail if I change `%NAME%` to `Geneious2019` in my override?

Reply:
@cwhits  thank you for your comment. You are absolutely right, it will break code signature verification.

I would, therefore, suggest introducing a new variable called `APPNAME` which allows us to define the name of the application in the dmg file if needed. That way we stay backwards compatible as old recipes do not override this variable and it is set to Geneious by default. New recipes for the 2019 version can then override the variable with `Geneious Prime`.

Reply:
See dadf079 for suggested changes.

Reply:
The need for %APPNAME% seem unnecessary. I successfully tested with 

```suggestion
				<string>%pathname%/Geneious*.app</string>
```

Reply:
@n8felton thanks for the suggestion! This solution is, of course, way cleaner. Please see the last change I made.